staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -388,26 +388,26 @@ func (s *store) GetToList(ctx context.Context, key string, resourceVersion strin
 	if err != nil {
 		return err
 	}
-	key = path.Join(s.pathPrefix, key)
-
-	getResp, err := s.client.KV.Get(ctx, key, s.getOps...)
-	if err != nil {
-		return err
-	}
-	if len(getResp.Kvs) == 0 {
```

Comment:
I don't agree with this comment - our goal shouldn't be having as small diff as possible.
I really think that with this change the code is much more understandable.

Reply:
Yes - I think we want to backport it.
but, it won't generate conflict on 1.10 and 1.9 and I think it's more important to have this code cleaner than to avoid conflict in 1.8. WDYT?

Reply:
cleaner is in the eye of the beholder, but I don't feel that strongly

Reply:
Yes - that's true.
But if you don't feel very strongly about this, I would prefer leaving as is.